capstone-frontend/src/components/AdminClubPage.jsx

```diff
@@ -27,63 +42,73 @@ class AdminClubPage extends Component {
     let data = {};
     const formElements = document.getElementById("update-club-form").elements;
     for (let i = 0; i < formElements.length; i++) {
-      if (formElements[i].name.length !== 0 && 
-          formElements[i].type !== "submit" && formElements[i].value.length !== 0) {
+      if (formElements[i].name.length !== 0 &&
+        formElements[i].type !== "submit" && formElements[i].value.length !== 0) {
         data[formElements[i].name] = formElements[i].value;
       }
     }
     data.id = this.state.club.id;
     if (window.localStorage.getItem("userID") !== this.state.club.ownerID) {
       alert("Update failed. You do not own this club.");
       return;
-    } 
+    }
 
-    fetch("/api/clubs", {method: "put", body: JSON.stringify(data)})
-        .then(function() {
-          history.push(`/clubpage/${data.id}`);
-        })
-        .catch(function(e) {
-          console.log(e);
-          alert("Looks like we're having trouble connecting to our database, hang tight!");
-        });  
+    fetch("/api/clubs", { method: "put", body: JSON.stringify(data) })
+      .then(function () {
+        history.push(`/clubpage/${data.id}`);
+      })
+      .catch(function (e) {
+        console.log(e);
+        alert("Looks like we're having trouble connecting to our database, hang tight!");
+      });
   }
 
   handleDelete = () => {
     if (window.localStorage.getItem("userID") !== this.state.club.ownerID) {
       alert("Delete failed. You do not own this club.");
       return;
-    } 
+    }
     const history = this.props.history;
-    fetch(`/api/clubs?id=${this.props.match.params.id}`, {method: "delete"})
-        .then(function() {
-            history.push("/myclubs");
-        })
-        .catch(function(err) {
-            //TODO #61: Centralize error output
-            alert(err);
-        });
+    fetch(`/api/clubs?id=${this.props.match.params.id}`, { method: "delete" })
+      .then(function () {
+        history.push("/myclubs");
+      })
+      .catch(function (err) {
+        //TODO #61: Centralize error output
+        alert(err);
+      });
   }
 
   componentDidMount() {
-    this.fetchClub();
+    this.fetchData();
   }
 
   render() {
+    const requesters = this.state.requesters.map(m =>
+        <UserCard
+          key={m.id}
+          user={m}
+          club={this.state.club}
+          fetchData={this.fetchData} />);
```

Comment:
I think it would be better to use the `map` expression here directly in the returned JSX below instead of using the `requesters` const. That way the JSX in the source code follows the order of the elements on the page and a reader doesn't have to scroll up and down to see what would fit into that slot.

Reply:
Changed!